lib/spack/spack/ci.py

```diff
@@ -14,31 +18,36 @@
 
 import llnl.util.tty as tty
 
-import spack.environment as ev
+import spack
+import spack.binary_distribution as bindist
+import spack.cmd.buildcache as buildcache
 import spack.compilers as compilers
+import spack.config as cfg
+import spack.environment as ev
 from spack.dependency import all_deptypes
 from spack.error import SpackError
 import spack.hash_types as ht
+from spack.main import SpackCommand
+import spack.repo
 from spack.spec import Spec
 import spack.util.spack_yaml as syaml
+import spack.util.web as web_util
 
-description = "generate release build set as .gitlab-ci.yml"
-section = "build"
-level = "long"
```

Comment:
this should probably be in the `build packages` ("build") section --  the "creating packages" section is about creating `package.py` files.  Or the environments section.  I'm not really sure which.